react-components/src/components/Account/account.js

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ *
+ *    Copyright 2019 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+import React from 'react';
+
+import AccountTrigger from '../AccountTrigger';
+import AccountDropdown from '../AccountDropdown';
+
+const Account = () => {
```

Comment:
We should rename this to `AccountContainer` or something similar.

Reply:
Sure, no problem, shall I rename the component name I added to `ui.apps`?

Reply:
No, the AEM component can keep its name

Reply:
Hi @dplaton, the `<Account>` component has been renamed to `<AccountContainer>`